oracle/datadog_checks/oracle/oracle.py

```diff
@@ -101,7 +101,11 @@ def _fix_custom_queries(self):
                 if prefix.startswith(self.__NAMESPACE__ + '.'):
                     prefix = prefix[len(self.__NAMESPACE__) + 1 :]
                 for column in query.get('columns', []):
-                    if column.get('type') != 'tag':
+                    # if the column is supposed to be skipped
+                    if not column.get('type') and not column.get('name'):
+                        continue
+
+                    elif column.get('type') != 'tag':
```

Comment:
```suggestion
                    if column.get('type') != 'tag' and column.get('name'):
```
I find it easier with less branches to the if

Reply:
@hithwen what is the behaviour in your suggestion if we have defined 'name' but not 'type' column?
`column.get('type') != 'tag'` matches if it returns `None`?